### PR TITLE
Send `branch` and `slug` when skipping

### PR DESCRIPTION
**Pull request by tmeasday**
For https://github.com/chromaui/chromatic-cli/issues/351.

Relies on https://github.com/chromaui/chromatic/pull/5707 & https://github.com/chromaui/chromatic/pull/5732

### CODE REVIEW

**Comment by tmeasday** [comment]
The branch has now been merged + released, can we release this change?

File: bin-src/tasks/gitInfo.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ export const setGitInfo = async (ctx, task) => {
   if (matchesBranch(ctx.options.skip)) {
     transitionTo(skippingBuild)(ctx, task);
     // The SkipBuildMutation ensures the commit is tagged properly.
-    if (await ctx.client.runQuery(TesterSkipBuildMutation, { commit })) {
+    if (await ctx.client.runQuery(TesterSkipBuildMutation, { commit, branch })) {
```
Comment (ghengeveld) [inline bin-src/tasks/gitInfo.js:62]: We should also send `slug`.

**Review by ghengeveld — CHANGES_REQUESTED** [review]

File: bin-src/tasks/gitInfo.js [hunk 2]
```diff
@@ -23,8 +23,8 @@ import invalidChangedFiles from '../ui/messages/warnings/invalidChangedFiles';
 import isRebuild from '../ui/messages/warnings/isRebuild';
 
 const TesterSkipBuildMutation = `
-  mutation TesterSkipBuildMutation($commit: String!) {
-    skipBuild(commit: $commit)
+  mutation TesterSkipBuildMutation($commit: String!, $branch: String) {
+    skipBuild(commit: $commit, branch: $branch)
```
Comment (ghengeveld) [inline bin-src/tasks/gitInfo.js:27]: I think you intended to send $slug as well?

**Review by ghengeveld — CHANGES_REQUESTED** [review]

**Comment by tmeasday** [comment]
@ghengeveld what is the best way to test this? I pushed the upstream branch https://github.com/chromaui/chromatic/pull/5732 to staging

**Comment by ghengeveld** [comment]
> @ghengeveld what is the best way to test this? I pushed the upstream branch [chromaui/chromatic#5732](https://github.com/chromaui/chromatic/pull/5732) to staging

You'll need a PR from a fork and then trigger a skipBuild with that `ownerName:branchName` combo. The `--branch-name` flag supports that format (it will override the detected slug if you do). Then I guess you can check Loggly for `Succesfully synced commit status for UI Review` with `Skipped build.` as the description.

**Comment by tmeasday** [comment]
> You'll need a PR from a fork and then trigger a skipBuild with that ownerName:branchName combo. The --branch-name flag supports that format (it will override the detected slug if you do). Then I guess you can check Loggly for Succesfully synced commit status for UI Review with Skipped build. as the description.

Thanks, that's useful, but actually what I just meant is how can I test this branch of the CLI?

**Comment by tmeasday** [comment]
Awesome. Worked! Thanks @ghengeveld 

image

https://github.com/chromaui/shapes/pull/44

**Review by ghengeveld — APPROVED** [review]

**Comment by tmeasday** [comment]
I'll let you merge + release this @ghengeveld ?

**Pull request merged**